File: codewars/JavaScript/7kyu/Find the next perfect square!.js
function findNextSquare(sq) {
    const square_root = sq ** 0.5
    const modulus_1 = square_root % 1

    if (modulus_1 == 0) {
        const next_perfect_square = square_root + 1
        return next_perfect_square * next_perfect_square
    }

    return -1
}

File: codewars/Python/7kyu/Find the next perfect square!.py
def find_next_square(sq):
    # Return the next square if sq is a square, -1 otherwise
    square_root = sq ** 0.5
    modulus_1 = square_root % 1

    is_perfect_square = (modulus_1 == 0)
    if is_perfect_square:
        next_perfect_square = square_root + 1
        return next_perfect_square * next_perfect_square
    
    return -1